config/schedule.rb

```diff
@@ -7,3 +7,8 @@
 every :day, at: '3:00 am', roles: [:job_creator] do
   rake 'clear_expired_sessions'
 end
+
+every Figaro.env.queue_health_check_frequency_seconds.to_i, roles: [:job_creator] do
+  runner 'WorkerHealthChecker.check'
+  runner 'WorkerHealthChecker.enqueue_dummy_jobs'
+end
```

Comment:
Yeah, I agree, it's nice that Figaro is standalone but, in this case, all of the app secrets are put through Figaro so let's keep it consistent. 👍